### PR TITLE
Add Android support for serial number, IP, and MAC address.

### PR DESCRIPTION
**Pull request by passabilities**

### CODE REVIEW

File: android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java [hunk 1]
```diff
@@ -80,10 +84,15 @@ private Boolean isTablet() {
     constants.put("buildVersion", "not available");
     constants.put("buildNumber", 0);
 
+    WifiManager manager = (WifiManager) this.reactContext.getSystemService(Context.WIFI_SERVICE);
+    WifiInfo wifiInfo = manager.getConnectionInfo();
+    String ipAddress = Formatter.formatIpAddress(wifiInfo.getIpAddress());
+    String macAddress = wifiInfo.getMacAddress();
```
Comment (passabilities) [inline android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java]: Just remembered that these will change over time so will have to create a function that will be called for a callback.

**Comment by avishayhajbi** [comment]
i got an error while asking for macAddress.
(after using you code). @passabilities

**Comment by passabilities** [comment]
@avishayhajbi what was the error?

**Comment by fjefferson** [comment]
Please, commit only get serial number.

**Comment by passabilities** [comment]
@fjefferson what do you mean?

**Comment by ss44** [comment]
any luck getting this merged?

**Comment by passabilities** [comment]
@ss44 just waiting for approval. I just rebased my branch on top of master so feel free to use my branch in the interim.

**Comment by GantMan** [comment]
LGTOM 👍

**Pull request merged**